portals/publisher/source/src/app/components/Apis/Create/AsyncAPI/ApiCreateAsyncAPI.jsx

```diff
@@ -267,43 +278,77 @@ export default function ApiCreateAsyncAPI(props) {
+                                                    defaultMessage='Available Protocols'
+                                                />
+                                            </Typography>
+                                        </Grid>
+                                        <Grid item xs={12} md={6} lg={9}>
+                                            {apiInputs.asyncTransportProtocols.map((protocol) => (
+                                                <Chip
+                                                    key={protocol}
+                                                    label={upperCaseString(protocol)}
+                                                    style={{
+                                                        'font-size': 13,
+                                                        height: 20,
+                                                        marginRight: 5,
+                                                    }}
+                                                    color='primary'
+                                                />
+                                            ))}
+                                        </Grid>
                                     </>
                                 )}
-                                value={apiInputs.protocol}
-                                name='protocol'
-                                SelectProps={{
-                                    multiple: false,
-                                    renderValue: (selected) => (selected),
-                                }}
-                                margin='normal'
-                                variant='outlined'
-                                InputProps={{
-                                    id: 'itest-id-apipolicies-input',
-                                }}
-                                onChange={handleOnChangeForProtocol}
-                            >
-                                {protocols.map((protocol) => (
-                                    <MenuItem
-                                        dense
-                                        disableGutters={false}
-                                        id={protocol.name}
-                                        key={protocol.name}
-                                        value={protocol.displayName}
-                                    >
-                                        <ListItemText primary={protocol.displayName} secondary={protocol.description} />
-                                    </MenuItem>
-                                ))}
-                            </TextField>
+                            </Grid>
+                            {apiInputs.gatewayVendor === 'wso2' && (
+                                <TextField
+                                    fullWidth
+                                    select
+                                    label={(
+                                        <>
```

Comment:
Redundant tag

Reply:
This was requested by the enlist extension to make is it errorless